kubewarden/ui
# Add build and helm chart for Rancher extension

## Description

This adds the build and helm chart for loading in Rancher as an extension 

## Test

- Add `https://github.com/jordojordo/ui.git` as a repository in the local cluster
- Go to the Extensions page from the side-nav
- See Kubewarden listed and install
- Navigate to the local cluster and view the Kubewarden menu item in the side resources menu
